SECURITY.md

```diff
@@ -0,0 +1,5 @@
+Security Policy
+===============
+
+We have outlined our Security Policy in our documentation.
+Please see https://symfony.com/doc/current/contributing/code/security.html.
```

Comment:
Should this not rather use the short link https://symfony.com/security?

Reply:
indeed

Reply:
What about:

Our [Security Policy](https://symfony.com/security) is described in our official documentation.

Reply:
I was not aware of a short link for the security page. TIL.